hypothesis-python/src/hypothesis/provisional.py

```diff
@@ -180,3 +181,8 @@ def url_encode(s):
         paths,
         st.just("") | _url_fragments_strategy,
     )
+
+
+def realize(value):
+    context = current_build_context()
+    return context.data.provider.realize(value)
```

Comment:
Unconvinced that this is worth putting in our public API, even provisionally - anyone using it is going to be exposed to the deeper internals anyway.

Reply:
hmm, I think this is mostly true for now, but I do think we'll have to recommend `realize` publicly at some point post-api-stabilization for users who run into the same issues we did (storing symbolics post function lifetime). We can debate that later though, I've moved it to `data.py`.